extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -515,6 +516,7 @@ def from_dict(self, model):
         Args:
             model: A dict representing a StringClassifier.
         """
+        model = self._unpickle_json_serialization(model)
```

Comment:
I guess we can skip this for now and I think we need to sort out this LDAStringClassifier issue as soon as possible. I'll write a mail later detailing the things I have tried, the results I have achieved and the possible options for us.